src/nodes.coffee

```diff
@@ -1596,15 +1711,18 @@ exports.Assign = class Assign extends Base
 # has no *children* -- they're within the inner scope.
 exports.Code = class Code extends Base
   constructor: (params, body, tag) ->
+    super()
+
     @params      = params or []
     @body        = body or new Block
     @bound       = tag is 'boundfunc'
     @isGenerator = !!@body.contains (node) ->
       (node instanceof Op and node.isYield()) or node instanceof YieldReturn
+    @isMethod    = false
 
   children: ['params', 'body']
 
-  isStatement: -> !!@ctor
+  isStatement: -> !!@method
```

Comment:
I guess this isn't working correctly (`@method` vs. `@isMethod`) 😨  Not sure what will be broken...